docs/skopeo-inspect.1.md

```diff
@@ -77,6 +77,10 @@ The username to access the registry.
 
 The password to access the registry.
 
+**--no-tags**, **-n**=_bool_
+
+Do not list in the output, all tags available in the same repository. Defaults to `false` because of which the output will include all tags available.
```

Comment:
```suggestion
Do not list the available tags from the repository in the output. When invoked, the `RepoTags` array will be empty.  Defaults to `false`, which includes all available tags.
```

Reply:
Done. The updated version of this PR (almost) uses this text. The only minor change I additionally did was, instead of saying "When invoked...", I used "When `true`...". I hope that's fine.